Use latest IMDS API version in examples

boumenot opened the pull request:
When you are copying and pasting from the Web you want to have the latest IMDS API version in the text.  This will ensure the most complete data are returned.

azurecla commented:
Hi __@onewth__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@pablissima 
FYI

rmca14 commented:
Mail sent to add **onewth** as a MS FTE GitHub contributors.

boumenot committed:
Use latest IMDS API version in examples

When you are copying and pasting from the Web you want to have the latest IMDS API version in the text.  This will ensure the most complete data are returned.

cloudbooster reviewed (approved):
Looks good, the reason why the version was kept older as that is what is rolled out in National clouds, but this is good.
